packages/dataviews/src/bulk-actions.tsx

```diff
@@ -21,19 +22,55 @@ const {
 	DropdownMenuSeparatorV2: DropdownMenuSeparator,
 } = unlock( componentsPrivateApis );
 
-export function useHasAPossibleBulkAction( actions, item ) {
+interface ActionWithModalProps {
+	action: ActionModal;
+	selectedItems: Item[];
+	setActionWithModal: ( action?: ActionModal ) => void;
+	onMenuOpenChange: ( isOpen: boolean ) => void;
+}
+
+interface BulkActionsItemProps {
+	action: Action;
+	selectedItems: Item[];
+	setActionWithModal: ( action?: ActionModal ) => void;
+}
+
+interface ActionsMenuGroupProps {
+	actions: Action[];
+	selectedItems: Item[];
+	setActionWithModal: ( action?: ActionModal ) => void;
+}
+
+interface BulkActionsProps {
+	data: Data;
+	actions: Action[];
+	selection: string[];
+	onSelectionChange: ( selection: Item[] ) => void;
+	getItemId: ( item: Item ) => string;
+}
+
+export function useHasAPossibleBulkAction( actions: Action[], item: Item ) {
 	return useMemo( () => {
 		return actions.some( ( action ) => {
-			return action.supportsBulk && action.isEligible( item );
+			return (
+				action.supportsBulk &&
+				( ! action.isEligible || action.isEligible( item ) )
```

Comment:
isEligible is not mandatory in actions (you can check that in other components). So typing this component actually highlights that this case was not being considered. So this should be a fix.

Reply:
Good catch 👍